src/sentry/api/serializers/rest_framework/dashboard.py

```diff
@@ -323,7 +323,7 @@ def validate(self, data):
 
     def update_dashboard_filters(self, instance, validated_data):
         page_filter_keys = ["environment", "period", "start", "end", "utc"]
-        dashboard_filter_keys = ["release"]
+        dashboard_filter_keys = ["release", "release_id"]
```

Comment:
Is it intentional this is `release_id` instead of `releaseId`?

Reply:
Yeah, it's intentional because the `CamelSnakeSerializer` converts it into snake case. I figured it would be good to treat it as such in the backend until we expect it back in a response, where I'm using `camel_to_snake_case` to index the filters

Does this seem reasonable to you? Or do you think I should re-convert it back into camel case before saving it?

Reply:
Nope that's fine :+1: Just wanted to make sure it was intentional, its been a while since i've touched serializers so wanted to check